Team6220_PowerPlay/src/main/java/org/firstinspires/ftc/team6220_PowerPlay/BaseOpMode.java

```diff
@@ -1,113 +1,140 @@
         motorBL.setDirection(DcMotorEx.Direction.FORWARD);
         motorBR.setDirection(DcMotorEx.Direction.REVERSE);
-        motorLeftSlides.setDirection(DcMotorEx.Direction.FORWARD);
-        motorRightSlides.setDirection(DcMotorEx.Direction.FORWARD);
-        motorTurntable.setDirection(DcMotorEx.Direction.FORWARD);
 
         motorFL.setMode(DcMotorEx.RunMode.STOP_AND_RESET_ENCODER);
         motorFR.setMode(DcMotorEx.RunMode.STOP_AND_RESET_ENCODER);
         motorBL.setMode(DcMotorEx.RunMode.STOP_AND_RESET_ENCODER);
         motorBR.setMode(DcMotorEx.RunMode.STOP_AND_RESET_ENCODER);
 
+        motorFL.setMode(DcMotorEx.RunMode.RUN_USING_ENCODER);
+        motorFR.setMode(DcMotorEx.RunMode.RUN_USING_ENCODER);
+        motorBL.setMode(DcMotorEx.RunMode.RUN_USING_ENCODER);
+        motorBR.setMode(DcMotorEx.RunMode.RUN_USING_ENCODER);
+
         motorFL.setZeroPowerBehavior(DcMotorEx.ZeroPowerBehavior.BRAKE);
         motorFR.setZeroPowerBehavior(DcMotorEx.ZeroPowerBehavior.BRAKE);
         motorBL.setZeroPowerBehavior(DcMotorEx.ZeroPowerBehavior.BRAKE);
         motorBR.setZeroPowerBehavior(DcMotorEx.ZeroPowerBehavior.BRAKE);
 
-        motorTurntable.setMode(DcMotorEx.RunMode.STOP_AND_RESET_ENCODER);
+        motorLeftSlides.setDirection(DcMotorEx.Direction.FORWARD);
+        motorRightSlides.setDirection(DcMotorEx.Direction.REVERSE);
+
         motorLeftSlides.setMode(DcMotorEx.RunMode.STOP_AND_RESET_ENCODER);
         motorRightSlides.setMode(DcMotorEx.RunMode.STOP_AND_RESET_ENCODER);
 
-        motorFL.setMode(DcMotorEx.RunMode.RUN_USING_ENCODER);
-        motorFR.setMode(DcMotorEx.RunMode.RUN_USING_ENCODER);
-        motorBL.setMode(DcMotorEx.RunMode.RUN_USING_ENCODER);
-        motorBR.setMode(DcMotorEx.RunMode.RUN_USING_ENCODER);
-
-        motorTurntable.setTargetPosition(0);
-        motorTurntable.setMode(DcMotorEx.RunMode.RUN_TO_POSITION);
-        motorTurntable.setZeroPowerBehavior(DcMotorEx.ZeroPowerBehavior.BRAKE);
-
         motorLeftSlides.setMode(DcMotorEx.RunMode.RUN_WITHOUT_ENCODER);
         motorRightSlides.setMode(DcMotorEx.RunMode.RUN_WITHOUT_ENCODER);
+
         motorLeftSlides.setZeroPowerBehavior(DcMotorEx.ZeroPowerBehavior.BRAKE);
         motorRightSlides.setZeroPowerBehavior(DcMotorEx.ZeroPowerBehavior.BRAKE);
 
         // servos
-        servoGrabber = hardwareMap.servo.get("servoGrabber");
+        PwmControl.PwmRange maxRange = new PwmControl.PwmRange(500, 2500, 20000);
+        servoGrabber = (ServoImplEx) hardwareMap.servo.get("servoGrabber");
+        servoGrabber.setPwmRange(maxRange);
 
         // initialize IMU
         BNO055IMU.Parameters parameters = new BNO055IMU.Parameters();
         parameters.angleUnit = BNO055IMU.AngleUnit.DEGREES;
+        parameters.accelUnit = BNO055IMU.AccelUnit.METERS_PERSEC_PERSEC;
         imu = hardwareMap.get(BNO055IMU.class, "imu");
         imu.initialize(parameters);
 
-        servoGrabber.setPosition(Constants.GRABBER_OPEN_POSITION);
+        sleep(3000);
```

Comment:
Do you really want a 3 second sleep in your initialization?

Reply:
This is likely a sleep that lets the camera initialize so we can detect the aprilTags

Reply:
No its to make sure the IMU has enough time to initialize correctly

Reply:
Since you're disagreeing on the reason, there should be a comment explaining why you're doing that

Reply:
added a comment